error LNK2019: unresolved external symbol

CommonLibSSE.lib(ExtraLevCreaModifier.cpp.obj) : error LNK2019: unresolved external symbol "public: virtual __cdecl RE::ExtraLevCreaModifier::~ExtraLevCreaModifier(void)" (??1ExtraLevCreaModifier@RE@@UEAA@XZ) referenced in function "public: virtual void * __cdecl RE::ExtraLevCreaModifier::`scalar deleting destructor'(unsigned int)" (??_GExtraLevCreaModifier@RE@@UEAAPEAXI@Z)

--- a/include/RE/E/ExtraLevCreaModifier.h
+++ b/include/RE/E/ExtraLevCreaModifier.h
@@ -23,7 +23,7 @@ namespace RE
 
 		ExtraLevCreaModifier();
 		explicit ExtraLevCreaModifier(LEV_CREA_MODIFIER a_modifier);
-		~ExtraLevCreaModifier() override;  // 00
+		~ExtraLevCreaModifier() override = default;  // 00
 
 		// override (BSExtraData)
 		ExtraDataType GetType() const override;                             // 01 - { return kLevCreaModifier; }

